box.json

```diff
@@ -52,14 +52,13 @@
         "format":"cfformat run system/**/*.cfc,tests/specs/**/*.cfc --overwrite",
         "format:watch":"cfformat watch system/**/*.cfc,tests/specs/**/*.cfc ./.cfformat.json",
         "format:check":"cfformat check system/**/*.cfc,tests/specs/**/*.cfc",
+        "log":"server log --follow",
+        "onServerInstall":"cfpm install zip,orm,mysql,postgresql,sqlserver,document,feed,mail,debugger",
         "start:lucee":"server start serverConfigFile='server-lucee@5.json' --force",
         "start:2018":"server start serverConfigFile='server-adobe@2018.json' --force",
         "start:2021":"server start serverConfigFile='server-adobe@2021.json' --force",
         "log:lucee":"server log coldbox-lucee@5 --follow",
         "log:2018":"server log coldbox-adobe@2018 --follow",
-        "log:2021":"server log coldbox-adobe@2021 --follow",
-        "cfpm":"echo '\".engine/acf2021/WEB-INF/cfusion/bin/cfpm.sh\"' | run",
```

Comment:
These are needed due to our location of the engines, you will have to restore them

Reply:
@lmajano It's not necessary to call the cfpm.sh script direct.  In fact, that script won't even work on Windows!  The CommandBox cfpm command will find the script in the server home for you, and is even better because it will ensure things like the `java_home` env var are set as well.

Reply:
My bad, looks like I accidentally dropped the `log:2021` script. I'll definitely restore that!

Reply:
Actually, this `log:2021` script is still there - just at the bottom of the scripts.